### PR TITLE
[NOTICE] Update NOTICE for 2022

### PR DESCRIPTION
**Pull request by pan3793**
### _Why are the changes needed?_

Welcome to 2022

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 Apache Kyuubi (Incubating)
-Copyright 2021 The Apache Software Foundation.
+Copyright 2021-2022 The Apache Software Foundation.
```
Comment (yaooqinn) [inline NOTICE:3]: using and onwards

File: NOTICE-binary [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 Apache Kyuubi (Incubating)
-Copyright 2021 The Apache Software Foundation.
+Copyright 2021-2022 The Apache Software Foundation.
```
Comment (yaooqinn) [inline NOTICE-binary:3]: ditto

File: NOTICE (hunk 1 above)
Comment (pan3793) [reply NOTICE:3]: We can not do that.
https://www.mail-archive.com/general@incubator.apache.org/msg75717.html

Comment (yaooqinn) [reply NOTICE:3]: thanks for sharing

**Review by yaooqinn — APPROVED** [review]
LGTM

**Comment by yaooqinn** [comment]
thanks merged to master, 1.4, 1.3
